assignment4.js

```diff
@@ -0,0 +1,16 @@
+var fs = require('fs');
+var fileArray= ['file1','file2','file3','file4','file5'];
```

Comment:
- there should be a space before and after operator in JS code.`a + b` but not `a+b`
- there should be a space after comma after each array element.